File: ui/src/main/java/com/midtrans/sdk/corekit/models/ExpiryModel.java
package com.midtrans.sdk.corekit.models;

public class ExpiryModel {
    public static final String UNIT_SECONDS = "seconds";
    public static final String UNIT_MINUTE = "minutes";
    public static final String UNIT_HOUR = "hours";
    public static final String UNIT_DAY = "days";

    private String startTime;
    private String unit;
    private int duration;

    public String getStartTime() {
        return startTime;
    }

    public void setStartTime(String startTime) {
        this.startTime = startTime;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }


}

File: corekit/src/main/kotlin/com/midtrans/sdk/corekit/api/model/ExpiryModel.kt
package com.midtrans.sdk.corekit.api.model

import com.google.gson.annotations.SerializedName

/**
 * Created by ziahaqi on 7/19/16.
 */
class ExpiryModel {
    @SerializedName("start_time")
    var startTime: String? = null

    @SerializedName("unit")
    var unit: String? = null

    @SerializedName("duration")
    var duration = 0

    companion object {
        const val UNIT_HOUR = "hours"
        const val UNIT_MINUTE = "minutes"
        const val UNIT_DAY = "days"
    }
}
